libsrc/atari/is_cmdline_dos.s

```diff
@@ -11,10 +11,9 @@
         .include "atari.inc"
 
 __is_cmdline_dos:
-        ldx     #0
         lda     __dos_type
         cmp     #MAX_DOS_WITH_CMDLINE + 1
-        txa
+        lda     #0
         rol     a
         eor     #$01
```

Comment:
Can't we omit the eor, or am I missing something?

__is_cmdline_dos:
   lda #MAX_DOS_WITH_CMDLINE
   cmp __dos_type
   lda #0
   rol a
   rts